### DISCUSSION TITLE
Validation with dependency to other fields

### DISCUSSION THREAD

**Discussion by MattFromGer** [formkit/formkit#655] [General]
If I have, let's say, a form with an email input and a phone input, how can I implement a validation that checks if either one is required?
The requirement is to have a contact information, where the user can decide which one is used.

```
<FormKit type="tel" label="Phone" validation="?" />
<FormKit type="email" label="E-Mail" validation="?" />
```

**Comment by luan-nk-nguyen** [comment]
Hi @MattFromGer! We have one on the way in an upcoming release. However, in the meantime, you can do something like this: https://formkit.link/7b9c38cd5bcdcc8c6b1ef3a05d684f1c

Closing this as it's a duplicate of https://github.com/formkit/formkit/issues/546.

**Comment by MattFromGer** [comment]
Perfect, that it's already on the radar! 👍
And thank you for the fast reply and intermediate solution.
